### ISSUE TITLE
Openlayer map not working

### ISSUE THREAD

**Issue by Boorhin** [Boorhin/Beach-Clean-Bay#1]
Needs to be able to work, needs to query Overpass turbo properly

**Comment by dcslagel** [comment]
Is there an example of the call and the error message?

**Comment by Boorhin** [comment]
At the moment I am just trying to parse a query for beach in Overpass turbo via openlayer.
I just get a blank page, I don't think I get how it is supposed to work

**Comment by dcslagel** [comment]
@Boorhin ,  Pull-Request #2 will hopefully resolve this issue.

**Comment by Boorhin** [comment]
thanks for fixing this map display issue, an unfortunate copy-paste maybe :/ 
There is still work onto getting the turbopass query to work and display the beaches within the bbox area. It is basically the method commented out that I took from another example.

**Comment by dcslagel** [comment]
Yeah, the map display didn't look quite right after getting it to display.  I'm new to Openlayer/turbopass queries.  I'll take another look and see if I can understand the query more..
